wellcomecollection/wellcomecollection.org
# Upgrade `husky`

## What does this change?

Upgrades `husky` to its latest version (5 major versions up!) and change config as per new set-up rules.

## How to test

Run locally, run `yarn` and make a change to a file that will upset `eslint`, then try to commit that change. You should be stopped at the pre-commit stage.

## How can we measure success?

Latest versions yo. Seems to also remove a ton of dependencies?

## Have we considered potential risks?
N/A
